proto/patches/api.json

```
@@ -1,4 +1,30 @@
 {
+  "tags": [
+    {
+      "name": "Authentication",
+      "description": "Login and logout of the cluster"
```

non-blocking: While we're here, maybe
```suggestion
      "description": "Log in to and out of of the cluster"
```

---

done, but not with the repeating "of"s : )